pkg/compilers/osv/osv_aws.go

```diff
@@ -1,23 +1,36 @@
 package osv
 
 import (
+	"io/ioutil"
+	"os"
+
 	"github.com/emc-advanced-dev/pkg/errors"
 	"github.com/emc-advanced-dev/unik/pkg/types"
 )
 
-type OsvAwsCompiler struct {
-	OSvCompilerBase
-}
-
 const OSV_AWS_MEMORY = 1024
 
-func (osvCompiler *OsvAwsCompiler) CompileRawImage(params types.CompileImageParams) (_ *types.RawImage, err error) {
-	resultFile, err := osvCompiler.CreateImage(params, true)
+type AwsCompilerHelper struct {
+	CompilerHelperBase
+}
+
+func (b *AwsCompilerHelper) Convert(params ConvertParams) (*types.RawImage, error) {
+	// Convert to WMDK format.
+	resultFile, err := ioutil.TempFile("", "osv-boot.vmdk.")
```

Comment:
As we discussed in Slack, it's also not perfectly clear why this "conversion" is strictly necessary - it basically just renames the file.